docs/how-to-add-support-for-new-package-manager.md

```diff
@@ -0,0 +1,43 @@
+# How to add support for a new package manager
+
+The sections below outline the steps needed to add support for a new package manager to OSS Review Toolkit.
+
+## 1. Initial Questions / Requirements
+
+Before implementing one must first determined whether the package manager meets the minimal requirements to provide the data needed for OSS Review Toolkit to work.
```

Comment:
Before implementing one must first determined whether the package manager meets the minimal requirements to provide the data needed for OSS Review Toolkit to work. -> Before you start any implementation work,  determine whether the package manager meets the minimal requirements to provide the data needed for OSS Review Toolkit to work.

Reply:
Done